lib/openstax/aws/deployment_base.rb

```diff
@@ -1,5 +1,24 @@
 module OpenStax::Aws
   class DeploymentBase
+    class Status
+      def initialize(deployment)
+        @deployment = deployment
+      end
+
+      def stacks
+        @deployment.stacks
+      end
+
+      def events
+        stacks.map do |stack|
```

Comment:
Don't think `def stacks` needs to be accessible to others above, so would just write `@deployment.stacks` here.